ci-operator/step-registry/upi/conf/vsphere/ova/upi-conf-vsphere-ova-commands.sh

```diff
@@ -54,6 +54,7 @@ fi
 
 echo "$(date -u --rfc-3339=seconds) - Checking if RHCOS OVA needs to be downloaded from ${ova_url}..."
 
+vsphere_version=$(govc about -json | jq -r .About.Version | awk -F'.' '{print$1}')
```

Comment:
how about adding a whitespace between `print` and `$1`

Reply:
Updated